Tests fail with py-vapid 1.5.0

Tests fail with py-vapid 1.5.0:

```
======================================================================
FAIL: test_as_curl (test_webpush.WebpushTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/var/tmp/paludis/build/dev-python-pywebpush-1.10.1/work/PYTHON_ABIS/2.7/pywebpush-1.10.1/pywebpush/tests/test_webpush.py", line 310, in test_as_curl
    ok_(s in result, "missing: {}".format(s))
AssertionError: missing: -H "authorization: vapid 
>>  raise AssertionError('missing: -H "authorization: vapid ')
    

======================================================================
FAIL: test_send_vapid (test_webpush.WebpushTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/var/tmp/paludis/build/dev-python-pywebpush-1.10.1/work/PYTHON_ABIS/2.7/pywebpush-1.10.1/pywebpush/tests/test_webpush.py", line 172, in test_send_vapid
    ok_('vapid' in pheaders.get('authorization'))
AssertionError: None
>>  raise AssertionError(None)
```

while they run fine with py-vapid 1.7.0, probably the dependency requirement should be updated to reflect that a more recent py-vapid is required.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # I am terrible at keeping this up-to-date.
 
+## 1.10.2 (2020-04-11)
+bug: update min vapid requirement to 1.7.0
+
 ## 1.10.1 (2019-12-03)
 feat: use six.text_type instead of six.string_types
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
 cryptography>=2.6.1
 http-ece>=1.1.0
 requests>=2.21.0
-py-vapid>=1.5.0
+py-vapid>=1.7.0

--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@ import os
 from setuptools import find_packages, setup
 
 
-__version__ = "1.10.1"
+__version__ = "1.10.2"
 
 
 def read_from(file):

